R/utils.R

```diff
@@ -89,11 +89,11 @@ sync_remote_file <- function(full_url, output_file, ...) {
 #' @importFrom glue glue
 #' @importFrom dplyr tbl
 #' @importFrom dbplyr sql
+#' @importFrom glue glue_sql
 #' @return An SQL data frame
 #' @keywords internal
 read_parquet <- function(conn, path){
-    path_list <- paste0("'", path, "'") |> paste(collapse = ", ")
-    from_clause <- glue("FROM read_parquet([{path_list}], union_by_name = true)")  |> sql()
+    from_clause <- glue_sql("FROM read_parquet([{`path`*}], union_by_name=true)", .con=conn) |> sql()
```

Comment:
`glue_sql` already returns an SQL object
```suggestion
    from_clause <- glue_sql("FROM read_parquet([{`path`*}], union_by_name=true)", .con=conn)
```

Reply:
removing |> sql() generated an error in `db_query_fields.DBIConnection()`.
```
It looks like you tried to incorrectly use a table in a
schema as source.
```

Reply:
Ah okay, `glue_sql` seems to set a different SQL class.